### ISSUE TITLE
getTopicName should be public to be accessed

### ISSUE THREAD

**Issue by wipatrick** [smueller18/avro-builder#2]
AvroBuilder.getTopicName(SolarCircleWarning.class);

```sh
Error:(110, 20) java: getTopicName(java.lang.Class<? extends com.github.smueller18.avro.builder.AvroBuilder>) is not public in com.github.smueller18.avro.builder.AvroBuilder; cannot be accessed from outside package
```

**Comment by wipatrick** [comment]
Other than that I found a way to access the @KafkaTopic without using this method:

**SolarCircleWarning**:
```java
@Documentation("Warning event if the temperature of the solar circle is abnormal high. Condition: TSOC_1 > 93.0 or TSOC_2 > 93.0")
@KafkaTopic("topic.test")
public class SolarCircleWarning extends AbstractEvent {

    @Key
    @TimestampMillisType
    @Documentation("timestamp when condition fulfilled initially")
    private long first_exceed_time = 0;

    @Value
    private double tsoc_1;

    @Value
    private double tsoc_2;


    public SolarCircleWarning(long first_exceed_time, double tsoc_1, double tsoc_2) {
        this.first_exceed_time = first_exceed_time;
        this.tsoc_1 = tsoc_1;
        this.tsoc_2 = tsoc_2;
    }

    // TODO: move to static
    public SolarCircleWarning () {

    }

    public long getFirst_exceed_time() {
        return first_exceed_time;
    }

    public void setFirst_exceed_time(long first_exceed_time) {
        this.first_exceed_time = first_exceed_time;
    }

    public double getTsoc_1() {
        return tsoc_1;
    }

    public void setTsoc_1(double tsoc_1) {
        this.tsoc_1 = tsoc_1;
    }

    public double getTsoc_2() {
        return tsoc_2;
    }

    public void setTsoc_2(double tsoc_2) {
        this.tsoc_2 = tsoc_2;
    }


}
```
**FlinkUtils:**
```java
 public static void produceToKafka(DataStream<? extends AvroBuilder> dataStream) {
        
        System.out.println(dataStream.getType().getTypeClass().getAnnotation(KafkaTopic.class).value());
}
```

returns:
```sh
topic.test
```

**Comment by smueller18** [comment]
Yes, that's also possible. But `AvroBuilder.getTopicName()` should be preferably used, because a special exception is thrown if the annotation `KafkaTopic` is not set.
Closed by c648e86
